QgisModelBaker/libqgsprojectgen/db_factory/db_simple_factory.py

```diff
@@ -17,19 +17,30 @@
  ***************************************************************************/
 """
 from .db_factory import DbFactory
-from .pg_factory import PgFactory
-from .gpkg_factory import GpkgFactory
-from .mssql_factory import MssqlFactory
 from QgisModelBaker.libili2db.globals import DbIliMode
 
+available_databases = dict()
```

Comment:
```suggestion
available_database_factories = dict()
```

I think that would make line 58 a bit easier to read

Reply:
Done!